src/components/app/app.js

```diff
@@ -6,6 +6,11 @@ import { TransactionsComponent } from "../transactions/transactions";
 import { AddAccountComponent } from "../add-account-dialog/add-account-dialog";
 import { AddTagComponent } from "../add-tag-dialog/add-tag-dialog";
 import { AboutComponent } from "../about-dialog/about-dialog";
+import { PieChartComponent } from "../pie-chart/pie-chart";
+import { BarChartComponent } from "../bar-chart/bar-chart";
+import { TableTransactionsComponent } from "../table-transactions/table-transactions";
+import { StorageService } from "../../services/storage";
+import { Account } from "../../services/model/account";
```

Comment:
why do you import Account in App, Add account dialog, Accounts and Drawer? (4 times) Wouldn't it be better to work with it in one place?

Reply:
cancel this question